### ISSUE TITLE
[Bug]: Invalid column type Text at index: 6, name: encrypted_value

### ISSUE THREAD

**Issue by majinke** [thewh1teagle/rookie#55]
### What happened?

today, i updated edge, then report "i updated edge", it's ok yesterday.
edge version: 130.0.2849.46 (64 bits)

### Steps to reproduce

1. step one...
2. step two...

**Comment by JoeanAmier** [comment]
I also encountered this problem.
```
import rookiepy

print(rookiepy.edge(domains=["douyin.com"]))
```

Unlocking Chrome database... This may take a while (sometimes up to a minute)
Traceback (most recent call last):
  File "C:\Users\youyq\AppData\Roaming\JetBrains\PyCharm2024.2\scratches\scratch_2.py", line 3, in <module>
    print(rookiepy.edge(domains=["douyin.com"]))
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Invalid column type Text at index: 6, name: encrypted_value

Location:
    rookie-rs\src\browser\chromium.rs:279:36

**Comment by git-070** [comment]
我也遇到了，不知道还能不能解决

**Comment by Saroumane** [comment]
Also reproduced since I upgraded chromium to version 130.0.6723.58

```
const cookies = chromium(["google.com"]);
                ^

Error: Invalid column type Text at index: 6, name: encrypted_value
```

I use : 
`import { chromium } from "@rookie-rs/api";`

**Comment by thewh1teagle** [comment]
See https://github.com/thewh1teagle/rookie/issues/50

Chrome updated the way it encrypts the cookies. 
I'm working on a fix but keep in mind that from now on in Windows it will be possible to decrypt the cookies ONLY when running as admin.

**Comment by thewh1teagle** [comment]
Released new version with support for latest chromium.
On Windows you must run it as admin (Python/JS/Rust)

**Comment by Saroumane** [comment]
Thanks, working again perfectly on Linux

**Comment by git-070** [comment]
> Released new version with support for latest chromium. On Windows you must run it as admin (Python/JS/Rust)
Hello! Thank you for the update

But the operation failed, what is the reason?
RuntimeError: key[0] !=  1
![RuntimeError key 0  != 1](https://github.com/user-attachments/assets/3a8d6202-1792-4d0f-bb66-d389e2dbb204)

**Comment by thewh1teagle** [comment]
> **Hello! Thank you for the update**
> But it failed to run, what is the reason?
> RuntimeError: key[0] != 1

It doesn't work with Edge or Brave yet.
It worked to me in Chrome and Chromium

**Comment by git-070** [comment]
> It doesn't work with Edge or Brave yet. It worked to me in Chrome and Chromium

**Understood, thank you!**

**Comment by thewh1teagle** [comment]
Fixed in v0.5.5. it works now on all Chrome based browsers. (Chrome/Edge/Brave etc)
